### PR TITLE
add new-arrival-item

### LINKED ISSUE #5: Component > new arrival item
**Mohammed**

this is the new arrival item in the home page, it should receive props `title`, `price`, `image`, `id`

image

### PR DESCRIPTION
**Pull request by Mohamad-Sabaa**
Feat: new arrival item 
Hello, this resolves #5

### CODE REVIEW

File: src/components/ArrivalItems/ItemList.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Item from "./Item";
+import Image from "../../images/Image";
+
+const data = {
+    id: 1,
+    img: {Image},
```
Comment (osamaakb) [inline src/components/ArrivalItems/ItemList.jsx:7]: image should be a string

File: src/components/ArrivalItems/ItemList.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    title: "Random Product",
+    price: "--$",
+};
+
+const ItemList = () => {
```
Comment (osamaakb) [inline src/components/ArrivalItems/ItemList.jsx:12]: no need for the `ItemList` component if you are only rendering one Item

File: src/components/ArrivalItems/Item.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    
+    
+    return (
+        <div className="mt-8">
+
+           <div className="container max-w-sm ">
+
+            <div id={id}>
+                    <div className="mx-14 ...">{Image}</div>
```
Comment (osamaakb) [inline src/components/ArrivalItems/Item.jsx:17]: what is the `...` dots for?

File: src/components/ArrivalItems/Item.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                      <div className="flex space-x-20 ...">
```
Comment (osamaakb) [inline src/components/ArrivalItems/Item.jsx:18]: same here

File: src/components/ArrivalItems/Item.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                          <h1 className="text-xl mr-6 text-olivegreen font-regular, mx-16">{title}</h1>
+                          <h3 className="text-xl mr-6 text-olivegreen font-regular mr-16">{price}</h3>
```
Comment (osamaakb) [inline src/components/ArrivalItems/Item.jsx:20]: you can't use `mr` twice

File: src/components/ArrivalItems/Item.jsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        
+            </div>
+          </div>
+        </div>
+    );
+};
+
+    Item.propTypes = {
+        id: PropTypes.number,
+        Image: PropTypes.func,
```
Comment (osamaakb) [inline src/components/ArrivalItems/Item.jsx:32]: this should `string`

**Review by osamaakb — CHANGES_REQUESTED** [review]
the component isn't rendering as it is supposed to, the button not centered too
add more releastic image to showcase the component

image

**Review by Mohamad-Sabaa — COMMENTED** [review]
fix #5

**Review by Mohamad-Sabaa — COMMENTED** [review]
Remove: ArrivalItemsData from App.js

File: src/Components/ArrivalItems/ArrivalItemsData.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+import ArrivalItems from "./ArrivalItems";
+
+const data = {
+    id: 1,
+    img: "./Images/arrivalItem.png",
+    title: "Random Product",
+    price: "--$",
+};
+
+const ArrivalItemsData = () => {
```
Comment (osamaakb) [inline src/Components/ArrivalItems/ArrivalItemsData.jsx:11]: please remove this file

File: src/Components/ArrivalItems/ArrivalItems.jsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+
+
+
+
+const ArrivalItems = ( {id, Image, title, price} ) => {
```
Comment (osamaakb) [inline src/Components/ArrivalItems/ArrivalItems.jsx:8]: format the code before submitting

File: src/Components/Navbar/Navbar.jsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import MainLogo from "../../Icons/MainLogo";
+import CardIcon from "../../Icons/CardIcon";
+const Navbar = () => {
```
Comment (osamaakb) [inline src/Components/Navbar/Navbar.jsx:5]: this code shouldn't be here, please remove it

**Review by osamaakb — CHANGES_REQUESTED** [review]

File: src/Components/Navbar/Navbar.jsx (hunk 9 above)
Comment (Mohamad-Sabaa) [reply src/Components/Navbar/Navbar.jsx:5]: should I remove the Navbar component ? I didn't use it

**Review by osamaakb — APPROVED** [review]

**Pull request merged**